irodsadmin/tests/test_commands.py

```diff
@@ -278,55 +296,42 @@ def test_get_orphans_project(self):
             collection,
         )
         self.irods.collections.create(orphan_path)
-        self.assertListEqual(
-            irodsorphans.get_orphans(
-                self.irods,
-                self.irods_backend,
-                self.expected_collections,
-                [self.assay],
-            ),
-            [orphan_path],
+
+        # Capture stdout
+        out = io.StringIO()
+        sys.stdout = out
+        irodsorphans.get_orphans(
+            self.irods,
+            self.irods_backend,
+            self.expected_collections,
+            [self.assay],
         )
+        output = out.getvalue()
+        sys.stdout = sys.__stdout__
 
-    def test_get_orphans_assay_subs(self):
+        self.assertIn(orphan_path, output)
+
+    def test_get_output_assay_subs(self):
         """Test get_orphans() with orphan assay subcollections"""
         collection = 'UnexpectedCollection'
         orphan_path = '{}/{}'.format(
             self.irods_backend.get_path(self.assay), collection
         )
         self.irods.collections.create(orphan_path)
-        self.assertListEqual(
-            irodsorphans.get_orphans(
-                self.irods,
-                self.irods_backend,
-                self.expected_collections,
-                [self.assay],
-            ),
-            [orphan_path],
-        )
 
-    def test_get_output(self):
```

Comment:
In new implementation this test got redundant, as we combined producing and logging steps in one